packages/paste-website/src/pages/components/help-text/index.mdx

```diff
@@ -0,0 +1,375 @@
+- Use the passive voice for input errors to avoid placing blame on the user. For example, "A friendly name is required."
+- Use the active voice for system errors. For example, "Our systems are currently down. Please contact our support team."
+
+<LivePreview scope={{Input, Label, HelpText}} language="jsx">
+  {`<>
+  <Label htmlFor="email_error_example">Email address</Label>
+  <Input aria-describedby="email_error_help_text" id="email_error_example" value="this is not a vaild entry" name="email_error_example" type="email" placeholder="example@twilio.com" onChange="" hasError />
+  <HelpText id="email_error_help_text" variant="error">Please enter a valid email.</HelpText>
+</>`}
+</LivePreview>
+
+## When to use help text
+
+Use help text when users might need additional information to fill out a form field.
+
+<DoDont>
+  <Do title="Do" body="Use help text in composition with a form element." center>
+    <Box width="100%" padding="space60">
+      <Label htmlFor="email_do">Email address</Label>
+      <Input
+        aria-describedby="email_do_help_text"
+        id="email_do"
+        name="email_do"
+        type="email"
+        placeholder="example@twilio.com"
+        onChange=""
+      />
+      <HelpText id="email_do_help_text">Please enter a valid email.</HelpText>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t use help text as a stand-alone component." center>
+    <Box width="100%" padding="space60">
+      <HelpText id="email_do_help_text">You should instead style the Text component with design tokens.</HelpText>
+    </Box>
+  </Dont>
+</DoDont>
+<DoDont>
+  <Do title="Do" body="Use help text to help users prevent errors and fill out a form field correctly." center>
+    <Box width="100%" padding="space60">
+      <Label htmlFor="phone_number_do">Phone number</Label>
+      <Input
+        aria-describedby="phone_number_do_help_text"
+        id="phone_number_do"
+        name="hone_number_do"
+        type="phone-number"
+        placeholder="(415) 888-CATS"
+        onChange=""
+      />
+      <HelpText id="phone_number_do_help_text">Please enter a valid phone number.</HelpText>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don't use placeholder text for validation instructions." center>
+    <Box width="100%" padding="space60">
+      <Label htmlFor="phone_number_dont">Email address</Label>
+      <Input
+        id="phone_number_dont"
+        name="phone_number_dont"
+        type="phone-number"
+        placeholder="Please enter a valid phone number."
+        onChange=""
+      />
+    </Box>
+  </Dont>
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="If you limit the length of text entry, show a character counter and explain to users in help text why their entry is restricted."
+    center
+  >
+    <Box width="100%" padding="space60">
+      <Label htmlFor="limited_do">Job title (60 characters)</Label>
```

Comment:
Are we encouraging the "60 characters" warning be in the Label _and_ Help Text? Or would it make sense just in Help Text? This is a teeny nit so feel free to ignore